www/scipylib/building/windows.rst

```diff
@@ -5,374 +5,213 @@ Building From Source on Windows
+
+To obtain this compiler, it is necessary to download for their **Intel Parallel Studio XE** product,
+which can be trialed for 30 days, but it is currently unknown what will happen to the library and header
+files on your hard drive after that period has expired. To download, visit this page `here <https://software.intel.com/en-us/fortran-compilers>`__
+for more information. However, if you are a **student** or **educator**, this option is quite
+viable because Intel's academic license will provide you everything that you need **free of charge**.
+To register, visit this page `here <https://software.intel.com/en-us/qualify-for-free-software>`__, choose the
+appropriate option corresponding to you. Afterwards, click the link corresponding to **Intel Parallel Studio XE**
+and download. Note that this installation will require that you have the most up-to-date version of `Visual Studio`_.
+
+ATLAS and LAPACK
+################
+
+ATLAS_ is an optimized version of BLAS that is considered to be "portably efficient" according to its website. If you
+want to use this library, the easiest is to use this library in combination with Mingw-w64_. Precompiled libraries using
+this toolchain can be found `here <https://github.com/matthew-brett/np-wheel-builder/tree/master/atlas-builds>`__ in the
+folder corresponding to your architecture (32-bit or 64-bit). While this setup has been shown to build NumPy successfully,
+it is not known yet whether it can build SciPy.
+
+If you are so inclined to build ATLAS_ by hand, you **must** use Cygwin to build it because the library was explicitly
+designed for Unix environments. However, you can compile the library either the native **gcc** tools or with the **mingww-64**
+tool package that you can download with Cygwin_. Installations scripts can be found in the same location `here <https://github.com/matthew-brett/np-wheel-builder/tree/master/atlas-builds>`__.
+In the folder corresponding to your architecture, search for an **install_atlas** script, download the appropriate ZIP files
+`here <http://nipy.bic.berkeley.edu/scipy_installers/atlas_builds/>`__, fill in some of the variables with appropriate
+values corresponding to your directory structure (e.g. the **code_home** variable) and then run script. **Be forewarned
+though that this will take a very long time (around eight hours) to install**.
+
+Finally, it should be noted that ATLAS_, although open source, is not well optimized for Windows given its intended
+operating system environment and is therefore not very optimized for a Windows environment. Thus, if performance is of
+the utmost importance, ATLAS_ may not be the best choice of libraries.
+
+OpenBLAS and LAPACK
+###################
+
+OpenBLAS_ is an optimized version of BLAS that is currently used in languages like Julia_ by default. Besides being
+actively worked upon, it performs about as well as the Intel libraries discussed previously. Furthermore, it is also
```

Comment:
this will install cygwin linked versions of openblas which work well with cygwin's python but probably not python.org native python